### ISSUE TITLE
Troubleshooting Forced-Bleaching of Facial Hair

### ISSUE THREAD

**Issue by somefriggnidiot** [Everfall-Development/ef-issues#12]
Ran into the issue with Ainsworth that many have reported before, but never found a resolution to, so I decided to dig in a bit...

I created Ainsworth with a dark-brown beard. Every time I load in, his beard and eyebrows are blonde. Swapping outfits (either reapplying the same outfit, or changing to something else, or changing back to the original outfit from changing to something else) does not change it. To fix return to my set facial hair color, I have to be in the clothing modification menu and activate the hair customizations tab, then select the green checkmar. That's all.

This immediately produces 4 errors of the exact same contents:
```
[   1198266] [b2944_GTAProce]        CrBrowserMain/ https://ef_clothing/changePedValueColor [object Object] (@ef_clothing/web/build/assets/index-FHHB7VOk.js:619)
[   1198266] [b2944_GTAProce]        CrBrowserMain/ TypeError: Failed to fetch (@ef_clothing/web/build/assets/index-FHHB7VOk.js:619)
[   1198266] [b2944_GTAProce]        CrBrowserMain/ https://ef_clothing/changePedValueColor [object Object] (@ef_clothing/web/build/assets/index-FHHB7VOk.js:619)
[   1198266] [b2944_GTAProce]        CrBrowserMain/ TypeError: Failed to fetch (@ef_clothing/web/build/assets/index-FHHB7VOk.js:619)
[   1198266] [b2944_GTAProce]        CrBrowserMain/ https://ef_clothing/changePedValueColor [object Object] (@ef_clothing/web/build/assets/index-FHHB7VOk.js:619)
[   1198266] [b2944_GTAProce]        CrBrowserMain/ TypeError: Failed to fetch (@ef_clothing/web/build/assets/index-FHHB7VOk.js:619)
[   1198266] [b2944_GTAProce]        CrBrowserMain/ https://ef_clothing/changePedValueColor [object Object] (@ef_clothing/web/build/assets/index-FHHB7VOk.js:619)
[   1198266] [b2944_GTAProce]        CrBrowserMain/ TypeError: Failed to fetch (@ef_clothing/web/build/assets/index-FHHB7VOk.js:619)
```

I can then force the hair back to blonde by using `/refreshskin`, which produces the exact same output as above.

So TLDR...

## What's happening?
Facial hair is loading in as blonde when the skin is loaded.

## Workaround fix?
Go to a clothing store or plastic surgeon, click on the mustache icon, click on the green checkmark.

## How to recreate?
Load your character in, facial hair will likely be blonde. Upon changing hair color, use `/refreshskin` to reload the blonde hair.

## Hypothesis:
Some body hairstyles aren't saving the user-selected hair color properly and defaulting to a "basic blonde." The clothing menu either stores or loads its values differently, causing it to be set properly when changing, but reverts to a default when the skin is reloaded.
